[AOTI] Turn on the ABI-compatible mode as default

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #136534
* #137660

Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @chauhang @rec

desertfire committed:
Update

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update

[ghstack-poisoned]

desertfire committed:
Update

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

chenyang78 reviewed (changes requested):
Please take a look at the failing test. Thanks!

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

chenyang78 reviewed (approved):
LGTM. Thanks!

desertfire commented:
@pytorchbot merge

desertfire commented:
@pytorchbot revert -m "The dependent PR https://github.com/pytorch/pytorch/pull/137660 fails in fbcode" -c nosignal

desertfire committed:
Update on "[AOTI] Turn on the ABI-compatible mode as default"


Summary: Make AOTI generate ABI-compatible code as default for OSS.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang rec

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge
